style.css

```diff
@@ -701,3 +582,329 @@ li {
+    font-size: 1.2vw;
+    letter-spacing: 0.03em;
+    color: white;
+  }
+
+  .flex_reversed {
+    flex-direction: row-reverse !important;
+  }
+
+  .about_myself {
+    background-color: whitesmoke;
+    order: 4;
+  }
+
+  .about_me_container {
+    padding: 20vh 10vw;
+    gap: 4vh;
+    background-color: white;
+    border-top-right-radius: 80px;
+    flex: 1;
+    position: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .about_me {
+    gap: 2.5vh;
+    justify-content: space-between;
+  }
+
+  .about_me_header {
+    font-size: 250%;
+    line-height: 1.35em;
+  }
+
+  .about_me p {
+    font-size: 1vw;
+  }
+
+  .socials2 {
+    display: flex;
+    justify-content: space-around;
+    flex-direction: row;
+    width: 140.17px;
+    order: 4;
+  }
+
+  .connect_text2 {
+    gap: 8vh;
+  }
+
+  .connect_text2 h2 {
+    font-size: 1.2vw;
+  }
+
+  .resume_link {
+    align-items: center;
+    padding: 0.5vw;
+    border-radius: 4px;
+  }
+
+  .resume_link p a {
+    font-weight: 700;
+    font-size: 1vw;
+    line-height: 1.3em;
+  }
+
+  .language_header {
+    width: 40vw;
+  }
+
+  .language_header_minimized {
+    width: 40vw;
+    height: 7vh;
+    padding-top: 2vh;
+  }
+
+  #frameworks {
+    padding-bottom: 4vh;
+    padding-top: 6.5vh;
+  }
+
+  .language_header h2 {
+    font-size: 1.4vw;
+    line-height: 1.3em;
+  }
+
+  .btn2 {
+    padding: 0;
+  }
+
+  .sep {
+    width: 105%;
+    align-self: flex-start;
+  }
+
+  .language_list_container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    width: 100%;
+  }
+
+  .language {
+    width: 7vw;
+    height: fit-content;
+    padding: 0;
+    padding-bottom: 2vh;
+    background: #f7f7f9;
+  }
+
+  .lan {
+    display: flex;
+    flex-direction: column;
+    align-self: flex-start;
+    gap: 1.5vh;
+    padding: 2.5vh 0 0.5vh 0.8vw;
+  }
+
+  .lan > img {
+    width: 3vw;
+  }
+
+  .language_name {
+    font-size: 1vw;
+  }
+
+  .contact_section_container {
+    background: #6070ff;
+    padding: 12% 25%;
+    padding-bottom: 8%;
+    background-repeat: no-repeat;
+    background-position: top center;
+    background-size: 97% 80%;
+    background-position-y: 55%;
+    background-image: url(images/contact\ form\ background\ shapes@2x.png);
+  }
+
+  .contact_section h2 {
+    font-size: 200%;
+    display: flex;
+  }
+
+  .contact_text p {
+    font-size: 1.4vw;
+  }
+
+  .contact_section form {
+    width: 70%;
+  }
+
+  .contact_section form > * {
+    width: 100%;
+    border-radius: 2px;
+    padding: 1% 9% 1% 4.5%;
+    border: 0;
+  }
+
+  #name,
+  #email {
+    font-size: 1.4vw;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+  }
+
+  #msg {
+    font-size: 1.4vw;
+    line-height: 1.3em;
+  }
+
+  #form_submit {
+    width: 40%;
+    display: flex;
+    justify-content: center;
+    align-self: center;
+    font-size: 1.4vw;
+    border-radius: 8px;
+    padding: 2vh 0;
```

Comment:
- [ ] Please, consider adding interaction effects for this one too! 🤖 🛠️ 

![image](https://user-images.githubusercontent.com/77069011/200000609-a90640ad-aa9b-4f0b-a9c6-a11dee5e0694.png)